src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -244,13 +257,12 @@ List of services that create entities:
 - Azure VM Scalesets
 - Azure VPN Gateways
 
-
   </Collapser>
 </CollapserGroup>
 
-## Migrate to the Azure Monitor integration
+## Migrate to the Azure Monitor integration [#migrate]
 
-When migrating from individual Azure integration to the Azure Monitor integration, there are a few things to keep in mind:
+When migrating from our traditional Azure integrations to the Azure Monitor integration, there are a few things to keep in mind:
```

Comment:
I really dislike bulleted lists with only two bullets. Do you see an easy way to get around that here?